docker-compose.yml

```diff
@@ -1,6 +1,15 @@
 version: "3.9"
 
 services:
+  transaction-service:
+    build:
+        context: ./transaction-service
+    container_name: transaction-service
+    ports:
+      - "5000:5000"
+    environment:
+      - NODE_ENV=development
```

Comment:
missing for mogo_uri, and JWT_Token, please refer to order-service to see the different, also pull the latest change in main to this as this docker compose is out of date